index.html

```diff
@@ -120,6 +120,35 @@ <h3><img src="images/circle.png" alt="circle icon"> Skills</h3>
     </div>
   </div>
 </section>
+<section id="form-section">
+  <div class="form-paragraph"> 
+       <h2>
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h2>
+    </div> 
+    <div class="form">
+      <form action="https://formspree.io/f/xnqwbvpv" method="post" id="contact-form">
+        <fieldset>
+          <input class="inputs" type="text" name="full-name" id="fullname" maxlength="30" placeholder="Enter your name"
+            required />
+          <input class="inputs" type="email" name="email" id="email" placeholder="Email" required />
+  
+          <textarea name="comments" id="comments" maxlength="500" placeholder="Enter text here"></textarea>
```

Comment:
- [ ] As [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#project-requirements) suggest, all input fields should have the required attribute to activate the validation on submit. Therefore, i suggest you ensure that the `textarea` field also has the `required` attribute ☺️.